exchange-v2/contracts/LibOrder.sol

```diff
@@ -24,9 +24,14 @@ library LibOrder {
         bytes data;
     }
 
-    function calculateRemaining(Order memory order, uint fill) internal pure returns (uint makeValue, uint takeValue) {
-        takeValue = order.takeAsset.value.sub(fill);
-        makeValue = LibMath.safeGetPartialAmountFloor(order.makeAsset.value, order.takeAsset.value, takeValue);
+    function calculateRemaining(Order memory order, uint fill, bool isMakeFill) internal pure returns (uint makeValue, uint takeValue) {
+        if (!isMakeFill){
+            takeValue = order.takeAsset.value.sub(fill);
+            makeValue = LibMath.safeGetPartialAmountFloor(order.makeAsset.value, order.takeAsset.value, takeValue);
+        } else {
+            makeValue = order.makeAsset.value.sub(fill);
+            takeValue = LibMath.safeGetPartialAmountFloor(order.takeAsset.value, order.makeAsset.value, makeValue);
```

Comment:
probably, we need to use safeGetPartialAmountCeil. need to think about it

Reply:
decided not to fix